storage/store.go

```diff
@@ -180,16 +193,20 @@ func (e *NotBootstrappedError) Error() string {
 }
 
 // storeRangeIterator is an implementation of rangeIterator which
-// cycles through a store's rangesByKey slice.
+// cycles through a store's rangesByKey btree.
 type storeRangeIterator struct {
 	store     *Store
+	stopper   *util.Stopper
 	remaining int
 	index     int
+	ch        chan *Range // Next range is written to this channel.
```

Comment:
Thanks for the suggestions! Sure, will rework on this. I just kept the existing interface to avoid touching other files (especially scanner_test.go), but there is no other reason.